runtime/bcutil/j9bcu.tdf

```diff
@@ -273,7 +273,7 @@ TraceEvent=Trc_BCU_stringInternTableCreated NoEnv Overhead=1 Level=3 Template="B
 TraceException=Trc_BCU_stringInternTableCreationFailed NoEnv Overhead=1 Level=3 Template="BCU stringInternTableCreationFailed: nodeCount=%u"
 TraceEvent=Trc_BCU_stringInternTableNotCreated NoEnv Overhead=1 Level=3 Template="BCU stringInternTableNotCreated"
 
-TraceEvent=Trc_BCU_internalDefineClass_FullData Overhead=1 Level=7 Template="BCU Trc_BCU_internalDefineClass_FullData: data=%.*s, classLoader=%p"
+TraceEvent=Trc_BCU_internalDefineClass_FullData Overhead=1 Level=7 Template="BCU internalDefineClass_FullData: data=%.*s, classLoader=%p"
```

Comment:
I don't think it's worth the risk. This tracepoint has been around for many years, what if some tooling is looking for the original message?

Reply:
The only part of the trace message that is guaranteed to be stable is the prefix `j9bcu.198`. The [documented rules](https://github.com/eclipse-openj9/openj9/blob/master/doc/diagnostics/AddingTracepoints.md) only disallow changing the signature of the tracepoint.

Reply:
People don't always follow the documented rules. In my opinion the risk out weighs the benefit of the change.

Reply:
What's the value of documenting the rules if we're not going to expect them to be followed?

I won't pursue this further, but, in my opinion, one of the values of that documentation is to provide clarity about what freedoms we have to improve our code base.